docker/README.md

```diff
@@ -0,0 +1,130 @@
+# 🪐 Docker Image and Container for Validate Tool
+
+The docker image of Validate Tool provides an easy way to deploy the Validate Tool, without having to 
+worry about the Java/JDK versions installed and operating systems supported. Also, this helps to 
+easily use the Validate Tool in other execution environments such as docker compose and Amazon ECS.
+
+The only prerequisite to use the dockerized Validate Tool is the availability of docker software in the host computer.
+At the time of writing this, the Validate Tool was tested with Docker version 20.10.12.
+
+## 🏃 Steps to build the docker image of the Validate Tool
+
+1. Clone the GIT Repository containing the validate tool.
+```shell
+git clone https://github.com/NASA-PDS/validate.git
+```
+
+2. Update (if required) the following version in the `Dockerfile` with the required Validate Tool version.
+
+| Variable            | Description |
+| ------------------- | ------------|
+| validate_version     | The version of the Validate Tool release to be included in the docker image |
+
+```    
+# Set following argument with the required validate tool version
+ARG validate_version=3.0.3
+```
+
+3. Open a terminal and change the current working directory to `validate/docker`.
+
+4. Build the docker image as follows.
+
+```
+    docker image build -t nasapds/validate .
+```
+
+5. Test the `nasapds/validate` docker image as follows.
+
+```
+    docker container run nasapds/validate --help
+```
+The above command should display the help content of the Validate Tool.
+
+
+## 🏃 Steps to run a docker container of the Validate Tool
+
+The [Operation documentation](https://nasa-pds.github.io/validate/operate/index.html) of the Validate 
+tool provides a detailed guide on executing the Validate Tool.
+
+The same commandline arguments can be used to execute the docker container of the Validate Tool with 
+the following docker specific change.
+
+- Docker containers cannot directly see the files and directories in the host machine. Therefore, it is required to map 
+the files and directories as volumes, so the docker container can access them.
+- Read more about docker volumes at https://docs.docker.com/storage/volumes/ 
+
+For example:
+
+A PDS bundle can be validated with Validate Tool as follows as per the 
+[Operation documentation](https://nasa-pds.github.io/validate/operate/index.html) of the Validate tool.
+
+```shell
+% validate $HOME/pds/bundle -M checksum-manifest.txt -r validate-report.txt -R pds4.bundle
+```
+
+However, when using the dockerized Validate Tool, above command can be updated as follows.
+
+```shell
+docker container run --name validate \
```

Comment:
Can drop `--name validate` since this container is going to be `--rm`oved 😀

Reply:
done